scripts/mkcloud

```diff
@@ -1471,11 +1470,13 @@ function sanity_checks()
 
     if [[ " ${steplist[*]} " == *" batch "* ]] ; then
         need_scenario=1
-        if [[ -n $scenario ]] ; then
-            : ${scenariofile:="${mkcloud_dir}/scenarios/cloud$(getcloudver)/${scenario}"}
-            [[ ! -f $scenariofile ]] && complain 115 "Scenario file not found at $scenariofile"
-        else
-            complain 114 "Please set a scenario file for crowbar_batch with scenario=foo"
+        if [[ -z $scenario ]] ; then
+            complain 114 "Please set scenario for crowbar_batch"
```

Comment:
It seemed unclear and redundant to me, and it was also wrong (the `.yaml` was missing). But I'll improve it!